tests/integration/test_concurrent_agent_creation.py

```
@@ -0,0 +1,319 @@
+            time.sleep(0.1)
+
+            proc2 = subprocess.Popen(
+                ["bash", str(SCRIPT_PATH), "lock-test-2", "Second agent"],
+                cwd=tmp_path,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                text=True,
+            )
+
+            # Wait for both to complete
+            stdout1, stderr1 = proc1.communicate(timeout=60)
```

Ruff (0.14.14)

[warning] Unpacked variable `stdout1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)